docs/best-practices/continuous-integration.md

```diff
@@ -447,7 +447,7 @@ The extension can be configured to install the latest version of _fastlane_ or a
 ## More Information
 For more documentation, see the [Apple App Store](https://marketplace.visualstudio.com/items?itemName=ms-vsclient.app-store) in the Visual Studio Team Server marketplace.
 
-# Nevercode Integration for iOS
+# Nevercode Integration for iOS & Android
```

Comment:
Now that both platforms are supported, could we maybe move the "supported platforms" bit to the first paragraph?

Reply:
Yeah sure,  I will do that:)